app/components/progress_tab/daily_progress_component_v2.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class ProgressTab::DailyProgressComponentV2 < ApplicationComponent
+  include AssetsHelper
+  include ProgressTabHelper
+
+  # We use 29 here because we also show today, so its 30 days including today
+  DAYS_AGO = 29
+  DAY_FORMAT = ApplicationHelper::STANDARD_DATE_DISPLAY_FORMAT
```

Comment:
I _think_ this format is outdated, and we use [this one](https://github.com/simpledotorg/simple-server/blob/master/config/initializers/time_and_date_formats.rb#L20) instead in most places. Can you confirm with the design team and replace if necessary?